tools/Modules/RFC/RFC.psm1

```diff
@@ -0,0 +1,369 @@
+$script:repoBase = "https://api.github.com/repos"
+
+<#
+.SYNOPSIS
+Get the highest RFC number which is actually part of the repo
+
+.DESCRIPTION
+Get the highest RFC number which is actually part of the repo
+
+.EXAMPLE
+PS> get-maxrfcnumber
+36
+
+.NOTES
+
+#>
+
+function Get-MaxRFCNumber {
+    $repoOwner = "PowerShell"
+    $repoName = "PowerShell-RFC"
+    $repoFiles = Get-RepoFileList -repoOwner $repoOwner -repoName $repoName
+    $pattern = "RFC(?<number>\d\d\d\d)-"
+    $numbers = $repoFiles.Where({$_.path -cmatch $pattern}).Foreach({if ( $_ -match $pattern ) { [int]($matches.number)}}) | Sort-Object
+    $numbers[-1]
+}
+
+<#
+.SYNOPSIS
+Get the repo file information for the highest RFC
+
+.DESCRIPTION
+Get the repo file information for the highest RFC
+
+.EXAMPLE
+PS> get-maxrfc
+
+size  path                                            file_url
+----  ----                                            --------
+10393 2-Draft-Accepted/RFC0036-AdditionalTelemetry.md https://github.com/PowerShell/PowerShell-RFC/blob/master/2-Draft-Accepted/RFC0036-AdditionalTelemetry.md
+
+.NOTES
+General notes
+#>
+function Get-MaxRFC {
+    $maxNumber = Get-MaxRFCNumber
+    $rfcPattern = "RFC{0:0000}" -f $maxNumber
+    $RFC = (Get-RepoFileList -repoOwner PowerShell -repoName PowerShell-RFC).Where({$_.path -cmatch $rfcPattern})
```

Comment:
You have a mix of hard coded repo values but also passing them around.  For now, I would suggest just defining the repoOwner and repoName as script variables at the top and just use those everywhere.

Reply:
done